app/controllers/async_info_controller.rb

```diff
@@ -61,4 +61,17 @@ def user_cache_key
     #{current_user&.articles_count}__
     #{current_user&.blocking_others_count}__"
   end
+
+  def navigation_links
+    @navigation_links = Rails.cache.fetch("navigation_links", expires_in: NUMBER_OF_MINUTES_FOR_CACHE_EXPIRY.minutes) do
```

Comment:
This is a great first iteration, and I also think this request is a great candidate for _edge caching_, as well as even browser caching as an idea to explore.

I just want to note that that could be a worthwhile follow-up step with more of an SRE focus. CC: @maestromac @fdocr. We could approach the discussion with an eye towards identifying ways to clean up some of the sprawling edge caching logic we're currently maintaining.

Reply:
Thanks Ben, that sounds good. @maestromac or @fdocr I'd love to work with one of you on the caching aspect if possible. I'm curious to dip my toes further into the caching realm.

Reply:
Per our call, it's better to move fully to the view file and leverage Rails' view/collection caching but that can happen in the follow-up PR.